src/azul/__init__.py

```diff
@@ -1335,6 +1335,23 @@ def security_contact(self) -> Optional[dict[str]]:
             import json
             return json.loads(value)
 
+    @attr.s(frozen=True, kw_only=True, auto_attribs=True)
+    class SlackIntegration:
+        workspace_id: str
+        channel_id: str
+
+    @property
+    def slack_integration(self) -> Optional[SlackIntegration]:
+
+        # FIXME: Eliminate local import
+        #        https://github.com/DataBiosphere/azul/issues/3133
+        import json
+        slack_integration = self.environ['azul_slack_integration']
+        if slack_integration is None:
```

Comment:
I don't think this can ever be true. `environ` either contains a key and then the value is a string, or it doesn't in which the previous line would raise. Please test both branches of your `if` statement.